app/models/organization.rb

```diff
@@ -2,20 +2,19 @@ class Organization < ActiveRecord::Base
   has_many :organization_users
   has_many :users, through: :organization_users
   has_many :orders
+  has_many :addresses
+  accepts_nested_attributes_for :addresses, allow_destroy: true
   validates :name, uniqueness: true
 
-  # default_scope { order("upper(name)") }
+  before_save :add_county
 
-  before_save :add_county_lat_lon
-
-  def add_county_lat_lon
-    if changed_attributes.keys.include?("address")
+  def add_county
+    return unless county.nil? && addresses.first
```

Comment:
Unless with multiple conditions always confuses me... what do you think about `return if county.present? || addresses.empty?`?